MovieQuiz/Presentation/MovieQuizViewController.swift

```diff
@@ -4,9 +4,166 @@ final class MovieQuizViewController: UIViewController {
+            text: "Рейтинг этого фильма больше чем 6?",
+            correctAnswer: false)
+    ]
+    
+    @IBAction private func yesButtonClicked(_ sender: UIButton) {
+        let currentQuestion = questions[currentQuestionIndex]
+        let givenAnswer = true
+        showAnswerResult(isCorrect: givenAnswer == currentQuestion.correctAnswer)
+    }
+    
+    @IBAction private func noButtonClicked(_ sender: UIButton) {
+        let currentQuestion = questions[currentQuestionIndex]
+        let givenAnswer = false
+        showAnswerResult(isCorrect: givenAnswer == currentQuestion.correctAnswer)
+    }
+    
+    @IBOutlet private var imageView: UIImageView!
+    @IBOutlet private var textLabel: UILabel!
+    @IBOutlet private var counterLabel: UILabel!
+    
+    private func show(quiz step: QuizStepViewModel) {
+        // здесь мы заполняем нашу картинку, текст и счётчик данными
+        imageView.image = step.image
+        textLabel.text = step.question
+        counterLabel.text = step.questionNumber
+    }
+    
+    private func show(quiz result: QuizResultsViewModel) {
+        // здесь мы показываем результат прохождения квиза
+        // создаём объекты всплывающего окна
+        let alert = UIAlertController(title: result.title, // заголовок всплывающего окна
+                                      message: result.text, // текст во всплывающем окне
+                                      preferredStyle: .alert) // preferredStyle может быть .alert или .actionSheet
+        
+        // создаём для него кнопки с действиями
+        let action = UIAlertAction(title: result.buttonText, style: .default, handler: { _ in
+            print("Repeat button is clicked!")
+            
+            // скидываем счётчик правильных ответов
+            self.numberOfCorrectAnswers = 0
+            self.currentQuestionIndex = 0
+            
+            // заново показываем первый вопрос
+            let firstQuestion = self.questions[self.currentQuestionIndex]
+            let viewModel = self.convert(model: firstQuestion)
+            self.show(quiz: viewModel)
+        })
+        
+        // добавляем в алерт кнопки
+        alert.addAction(action)
+        
+        // показываем всплывающее окно
+        self.present(alert, animated: true, completion: nil)
+    }
+    
+    private func convert(model: QuizQuestion) -> QuizStepViewModel {
+        return QuizStepViewModel(
+            image: UIImage(named: model.image) ?? UIImage(), // распаковываем картинку
+            question: model.text, // берём текст вопроса
+            questionNumber: "\(currentQuestionIndex + 1)/\(questions.count)")// высчитываем номер вопроса
+    }
+    
+    private func showAnswerResult(isCorrect: Bool) {
+        imageView.layer.masksToBounds = true
+        imageView.layer.borderWidth = 8
+        imageView.layer.borderColor = isCorrect ? UIColor.ypGreen.cgColor : UIColor.ypRed.cgColor
+        if isCorrect {numberOfCorrectAnswers += 1}
```

Comment:
после/перед фигурной скобкой ставится либо пробел либо перенос строки :)
```suggestion
        if isCorrect { numberOfCorrectAnswers += 1 }
```